test/only.js

```diff
@@ -0,0 +1,13 @@
+QUnit.module( "QUnit.only" );
+
+QUnit.test( "implicitly skipped test", function( assert ) {
+	assert.ok( false, "test should be skipped" );
+});
+
+QUnit.only( "only run this test", function( assert ) {
+	assert.ok( true, "only this test should run" );
+});
+
+QUnit.test( "another implicitly skipped test", function( assert ) {
+	assert.ok( false, "test should be skipped" );
+});
```

Comment:
The current behavior is the last only wins. I'm happy to add a spec for that behavior. Making the first only win mean setting some state that I could interrogate later when the next only is encountered.